Account Onboard Utility/v10/README.md

```diff
@@ -15,6 +15,8 @@ The Tool supports three modes: [*Create*](#create-command), [*Update*](#update-c
 The tool will create a log file in the same folder of the script called: _"Account_Onboarding_Utility.log"_
 Running the tool with common parameters of Debug and Verbose will add more information to the log
 
+With the newer verson of the REST API's (now seen as 2nd gen in the CyberArk documentation), in order to be able to upload accounts that have custom file catergories these need to be already enabled/set at the platform level of the platform that the accounts will be linked with. This is also relevant to be able to upload accounts that have have linked 'login' and 'reconcile' accounts. If you try to upload accounts that have custom file catergories as columns in the csv a meaningful error will be seen relating to the FC not being linked with the platform. There are six FC's that needed to be added to the platform. three are for the linked 'login' account and three are for the linked 'reconcile' account. Further information on how to do this can be found in this CyberArk KB: https://cyberark-customers.force.com/s/article/Add-Reconcile-and-Login-Accounts-to-an-Account-using-V10-REST-API
```

Comment:
I recommend using "Platform Properties (File Categories)" instead of "FC"
Also replace "file categories" with account properties maybe
Just to make sense with easy and simple terminology
The specifics about reconcile and logon - good that you added the link to the solution (might worth making it a real link) and I think it is worth having its own header - WDYT?

Reply:
thanks, good points

Reply:
I just updated the file